app/controllers/HasSupplementaryUnitController.scala

```diff
@@ -106,11 +109,21 @@ class HasSupplementaryUnitController @Inject() (
         .bindFromRequest()
         .fold(
           formWithErrors => Future.successful(BadRequest(view(formWithErrors, mode, recordId, onSubmitAction))),
-          value =>
+          value => {
```

Comment:
so on the others this is in the submit - so i put it here too - but i think i should be in the onPageload what do you think?

Reply:
I say keep it here and open the discussion on  "Does it 'start' at the submit point or 'start' when the page loads" - Kara may have an opinion

Reply:
we might need some tech debt to change it in other situations then

Reply:
needs to be onPageLoad. Has moved to onPageLoad in other scenarios on a earlier ticket this morning